docs/release_notes/pending_release.rst

```diff
@@ -39,6 +39,8 @@ Miscellaneous
 * Add script to help with updating versioning and updating changelog during
   the release process
 
+* Updated README to include reference to the SMQTK-IQR package.
```

Comment:
More of a question than an issue, but I noticed the pending release notes files use both past and present tense (`Add` and `Updated`) versus all present (`Add` and `Update`) or all past tense (`Added` and `Updated`). Do you have a preference on using past vs. present tense in these files?

Reply:
Yeah, I've never been the best at this, nor at enforcing this with others. On this particular point I would say I'm preferential to the past tense, but I'm not overly married to it as I realize there are perspective justifications for both angles. I can add a card to our board to normalize the tenses used across change notes, or at least in the current pending files, and to add verbiage to the contributing file.